django_adobesign/client.py

```diff
@@ -244,6 +244,19 @@ def get_signer(self, agreement_id, signer_id):
         response.raise_for_status()
         return response.json()
 
+    @handle_adobe_exception
+    def update_signer(self, agreement_id, signer_id, participant):
+        """
+        Update and return the signer with the given signer_id who belongs to
+        the agreement corresponding to the agreement_id.
+        """
+        url = self.build_url('agreements/{}/members/participantSets/{}'
+                             .format(agreement_id, signer_id))
```

Comment:
could be formatted differently:

`url = self.build_url(
		'agreements/{}/members/participantSets/{}'.format(agreement_id, signer_id)
	)`

Reply:
You're right 👍 
But I tried to format the code as close as the existing code base